website/source/docs/platform/k8s/run.html.md

```diff
@@ -0,0 +1,171 @@
+---
+layout: "docs"
+page_title: "Running Vault - Kubernetes"
+sidebar_current: "docs-platform-k8s-run"
+description: |-
+  Vault can run directly on Kubernetes in various configurations.  For pure-Kubernetes worloads, this enables Vault to also exist purely within Kubernetes.
+---
+
+# Running Vault on Kubernetes
+
+Vault can run directly on Kubernetes in various modes: `dev,` `standalone` and `ha`.
+For pure-Kubernetes workloads, this enables Vault to also exist purely
+within Kubernetes.
+
+This page starts with a large how-to section for various specific tasks.
+
+## Helm Chart
+
+The [Vault Helm chart](https://github.com/hashicorp/vault-helm)
+is the recommended way to install and configure Vault on Kubernetes.
+In addition to running Vault itself, the Helm chart is the primary
+method for installing and configuring Vault to integrate with other
+services such as Consul for High Availability deployments.
+
+While the Helm chart exposes dozens of useful configurations and automatically
+sets up complex resources, it **does not automatically operate Vault.**
+You are still responsible for learning how to monitor, backup,
+upgrade, etc. the Vault cluster.
+
+The Helm chart has no required configuration and will install a Vault 
+cluster with sane defaults out of the box. Prior to going to production,
+it is highly recommended that you
+[learn about the configuration options](/docs/platform/k8s/helm.html#configuration-values-).
+
+~> **Security Warning:** By default, the chart will install an insecure configuration
```

Comment:
Do we want to copy/paste this to the above security section that was `TODO`?

Reply:
Done!